openfisca_us/variables/finance/benefit/inputs.py

```diff
@@ -0,0 +1,24 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+
+
+class gross_income(Variable):
+    value_type = float
+    entity = BenefitUnit
+    label = u"Income before SNAP program deductions are applied"
+    definition_period = YEAR
+
+
+class net_income(Variable):
+    value_type = float
+    entity = BenefitUnit
+    label = u"Income after SNAP program deductions are applied"
+    definition_period = YEAR
+
+
+class people(Variable):
```

Comment:
I don't think this is used so we could just take it out, but ideally `people` is person-level, so that `sim.calc("people", map_to="any_entity")` gives person counts